pkg/cmd/server/origin/asset.go

```diff
@@ -220,8 +233,14 @@ func (c *AssetConfig) addHandlers(mux *http.ServeMux) error {
 		KubernetesVersion: kVersionInfo.GitVersion,
 		OpenShiftVersion:  oVersionInfo.GitVersion,
 	}
+
+	extensionPropertyKeys, extensionPropertyValues := extensionPropertyArrays(c.Options.ExtensionProperties)
+	extensionProps := assets.WebConsoleExtensionProperties{
```

Comment:
working on a PR to switch to a single array

On Fri, May 20, 2016 at 11:24 AM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/server/origin/asset.go
> <https://github.com/openshift/origin/pull/8934#discussion_r64059225>:
>
> > @@ -220,8 +233,14 @@ func (c *AssetConfig) addHandlers(mux *http.ServeMux) error {
> >  		KubernetesVersion: kVersionInfo.GitVersion,
> >  		OpenShiftVersion:  oVersionInfo.GitVersion,
> >  	}
> > +
> > +	extensionPropertyKeys, extensionPropertyValues := extensionPropertyArrays(c.Options.ExtensionProperties)
> > +	extensionProps := assets.WebConsoleExtensionProperties{
>
> an array of WebConsoleExtensionProperty structs with Key/Value fields
> might be better than parallel arrays
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8934/files/21217f567be4714edada6e6301ec46d2e6ff706b#r64059225>
>